cfme/common/vm.py

```diff
@@ -522,20 +521,19 @@ class VM(BaseVM):
     TO_RETIRE = None
 
     retire_form = Form(fields=[
-        ('date_retire', {
-            version.LOWEST: date_retire_element,
-            "5.5": AngularCalendarInput(
-                "retirement_date", "//label[contains(normalize-space(.), 'Retirement Date')]")}),
+        ('date_retire', AngularCalendarInput(
+            "retirement_date", "//label[contains(normalize-space(.), 'Retirement')]")),
```

Comment:
I see what you mean, the label returned by that locator could be clickable? I could match on 'Retirement Date' but there's also no guarantee the UI won't be changed to make that specific label clickable?  I could set to some other text on the page, like the h3 summary?